src/store/useLabyrinthStore.jsx

```diff
@@ -0,0 +1,101 @@
+import { create } from 'zustand';
+import { persist } from 'zustand/middleware';
+
+export const useLabyrinthStore = create(
+  persist((set) => ({
```

Comment:
Very nice ⭐ Just make sure to reset the initial state when restarting the game so that I don't see the old actions I've taken